extensions/wikia/UserProfilePageV3/templates/UserProfilePage_WikiaMobileRenderUserIdentityBox.php

```diff
@@ -1,47 +1,46 @@
 <section class=mst>
 	<div class=mstHhdr>
-		<!-- <img src="<?= $wg->BlankImgURL ;?>" data-src="<?= $user['avatar']; ?>" itemprop=image class="lazy imgPlcHld"> --!>
-		<!-- <noscript><img src="<?= $user['avatar']; ?>" itemprop=image></noscript> -->
 		<div class=mstAva>
-			<img class=mstAvaImg src="<?= $user['avatar']; ?>" itemprop=image>
+			<img class=mstAvaImg src="<?= Sanitizer::encodeAttribute( $user['avatar'] ); ?>" itemprop=image>
 		</div>
 		<hgroup class=mstHgr>
 			<? if( !empty($user['name']) ): ?>
-				<h1 class=mstNm itemprop=name><?= $user['name']; ?></h1>
+				<h1 class=mstNm itemprop=name><?= htmlspecialchars( $user['name'] ); ?></h1>
 			<? endif; ?>
 			<? if( !empty($user['tags']) ): ?>
 				<?php foreach($user['tags'] as $tag): ?>
 					<span class=mstGr><?= $tag; ?></span>
 				<?php endforeach; ?>
 			<? endif; ?>
 			<? if(!empty($user['chatBanned'])): ?>
-				<span class=mstGr><?= wfMsg('user-identity-box-banned-from-chat'); ?></span>
+				<span class=mstGr><?= wfMessage( 'user-identity-box-banned-from-chat' )->escaped(); ?></span>
 			<? endif; ?>
 			<? if( !empty($user['realName']) ): ?>
-				<h2 class=mstRn><?= wfMsg('user-identity-box-aka-label', array('$1' => $user['realName']) ); ?></h2>
+				<h2 class=mstRn><?= wfMessage( 'user-identity-box-aka-label', htmlspecialchars( $user['realName'] ) )->plain(); ?></h2>
```

Comment:
`->plain()` should be avoided where possible, so `->parse()` or `->escaped()` should be used. Since these have `<span>` in them, I'd use `->parse()` and pass the param as a `rawParam` (with `htmlspecialchars` still there) since we probably don't want the param parsed as wikitext. The same throughout for all the `->plain()` instances.